### DISCUSSION TITLE
Is there any string a remote session can send that will make Tabby produce a sound and raise a notification?

### DISCUSSION THREAD

**Discussion by tv21** [Eugeny/tabby#9881] [Q&A] [answered]
For example, if you are using iTerm on a Mac and on an ssh session you enter this at a command prompt:

echo $'\e]9;Task is finished!\007'

It will sound a bell sound and raise a system notification bubble with the text "Task is finished!" (of course you can change the text).  Tabby just ignores this, but is there any equivalent in Tabby (particularly in the Linux version)?

**Comment by Eugeny** [comment, marked as answer]
Looks like the 3rd party [osc-notify plugin](https://www.npmjs.com/package/tabby-osc-notify) already implements OSC9 notifications

**Reply by tv21** [reply]
Thanks, that works great!
